drift-codec/src/test/java/com/facebook/drift/codec/AbstractThriftCodecManagerTest.java

```diff
@@ -292,6 +293,14 @@ public void testArrays()
         testRoundTripSerialize(arrayField, TCompactProtocol::new);
     }
 
+    @Test
```

Comment:
Can you add some more tests for invalid inputs, and more parts of the URI?

Reply:
will do

Reply:
done

Reply:
done